File: Arrays/Practice/remove_duplicates.c
#include <stdio.h>

int main() {

    int arr[] = {10, 10, 20, 20, 30, 30};
    int n = 6;

    int newSize = 1;

    for (int i = 1; i < n; i++) {

        if (arr[i] != arr[newSize - 1]) {
            arr[newSize] = arr[i];
            newSize++;
        }
    }

    printf("Array after removing duplicates:\n");

    for (int i = 0; i < newSize; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}

File: Arrays/Practice/remove_duplicates.java
package Practice;
public class remove_duplicates {

    public static void main(String[] args) {

        int[] arr = {10, 10, 20, 20, 30, 30};

        int newSize = 1;

        for (int i = 1; i < arr.length; i++) {

            if (arr[i] != arr[newSize - 1]) {
                arr[newSize] = arr[i];
                newSize++;
            }
        }

        System.out.println("Array after removing duplicates:");

        for (int i = 0; i < newSize; i++) {
            System.out.print(arr[i] + " ");
        }
    }
}